### ISSUE TITLE
Error when using make on Windows

### ISSUE THREAD

**Issue by BenStigsen** [wren-lang/wren#704]
Followed the instructions on the **Getting Started** page on wren.io. I got the following error:
```
mingw32-make[1]: Entering directory 'C:/wren'
process_begin: CreateProcess(NULL, printf "%10s %-30s %s\n" run util/build_libuv.py, ...) failed.
make (e=2): The system cannot find the file specified.
mingw32-make[1]: *** [util/wren.mk:241: build/libuv.a] Error 2
mingw32-make[1]: Leaving directory 'C:/wren'
mingw32-make: *** [Makefile:16: release] Error 2```

**Comment by iwillspeak** [comment]
Do you have a Python interpreter installed? It is needed to build on of Wren's dependencies.

**Comment by underscorediscovery** [comment]
There are various issues in the steps on Windows. [I'm working on fixing those with the 0.3.0 stuff](http://wren.io/blog/0.2.0-and-beyond.html), so for now it's better to hold off briefly on trying to fix master.

**Comment by pyway** [comment]
On windows 32/64, there were some issues need to be corrected , such as file  path separator, included header files ...  @underscorediscovery

**Comment by underscorediscovery** [comment]
yes I stated that right before :) They'll be addressed.
 > There are various issues in the steps on Windows.

**Comment by ruby0x1** [comment]
Fixed in 0.3.0, use the visual studio project for a seamless experience.
http://wren.io/getting-started.html
